schema.prisma

```diff
@@ -156,7 +156,7 @@ model Campaign {
   slug                    String           @unique @db.VarChar(250)
   title                   String           @db.VarChar(200)
   essence                 String           @db.VarChar(500)
-  bankHash                String           @map("bank_hash") @db.VarChar(5)
+  paymentReference        String           @map("payment_reference") @unique @db.VarChar(15)
```

Comment:
You've picked a more suitable and appropriate name for the field, thanks!

Uniqueness of the field is a must-have :+1: